tool/tctl/common/auth_command.go

```diff
@@ -81,6 +84,9 @@ type AuthCommand struct {
 	authExport   *kingpin.CmdClause
 	authSign     *kingpin.CmdClause
 	authRotate   *kingpin.CmdClause
+	authLS       *kingpin.CmdClause
+
+	stdout io.Writer
```

Comment:
I'm assuming this is meant for testing, but I don't see any test. If we are going to introduce this I think that it should be used for every command, otherwise it's going to be confusing. Could do that in a separate PR.

Reply:
If that is the case, we could consider adding it as a parameter on the [`TryRun`](https://github.com/gravitational/teleport/blob/master/tool/tctl/common/tctl.go#L85). But I agree it should be handled in a separate PR if we do this.

Reply:
So for this I copied what already existed in `resource_command.go`, with the intent to finish a test for it, but I didn't get around to finishing the test before I had some family issues to attend to. In this particular case, I can remove it and look at adding another PR that covers the tests cases and adds a parameter to `TryRun`